packages/manager/modules/pci/src/projects/new/payment/credits/index.js

```diff
@@ -0,0 +1,18 @@
+import angular from 'angular';
+import '@ovh-ux/ng-translate-async-loader';
+import 'angular-translate';
+
+import component from './credits.component';
+
+const moduleName = 'ovhManagerPciProjectsNewPaymentCredits';
+
+angular
+  .module(moduleName, [
+    'oui',
+    'ngTranslateAsyncLoader',
+    'pascalprecht.translate',
+  ])
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .component('pciProjectNewPaymentCredits', component);
```

Comment:
Prefer `.component` before `.run`

Reply:
Is there a reason to set `.component` before `.run`?
I try to define them in their lifecycle order.